libs/core/src/lib/text/text.component.ts

```diff
@@ -0,0 +1,91 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    Input,
+    ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+    selector: 'fd-text',
+    templateUrl: './text.component.html',
+    styleUrls: ['./text.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TextComponent {
+    /**
+     * Text for render
+     */
+    @Input()
+    text: string;
+
+    /**
+     * Max visible lines of text
+     */
+    @Input()
+    maxLines = null;
+
+    /**
+     * Property allowing browsers to render whitespace and tabs
+     */
+    @Input()
+    whitespaces?: boolean;
+
+    /**
+     * Property for managing hyphenation, using css rule hyphens. Values: none | manual | auto
+     */
+    @Input()
+    hyphenation?: string;
+
+    /**
+     * Option that adds more and less buttons to expand/collapse text
+     */
+    @Input()
+    expandable?: boolean;
```

Comment:
Maybe I missed a discussion on this subject, but our standard practice here is to simply set a value and the type is automatically inferred by the docs/compiler.  So you'd just have `expandable = false`

Reply:
Good point, thank you.

Reply:
Changed